cmake/kconfig.cmake

```diff
@@ -228,6 +228,10 @@ if(EXISTS ${BOARD_DEFCONFIG})
 
         # platform-specific include path
         include_directories(${PX4_SOURCE_DIR}/platforms/${PX4_PLATFORM}/src/px4/common/include)
+
+        if(PLATFORM STREQUAL "qurt")
+            include(${PX4_SOURCE_DIR}/boards/modalai/voxl2/cmake/voxl2_qurt.cmake)
```

Comment:
Isn't this basically all a toolchain anyway?

voxl2_qurt.cmake + qurt_reqs.cmake + Toolchain-qurt.cmake => CMAKE_TOOLCHAIN_FILE?

See if(TOOLCHAIN) below on line 241.